eng/common/pipelines/templates/jobs/generate-job-matrix.yml

```diff
@@ -42,6 +42,11 @@ parameters:
 - name: PreGenerationSteps
   type: stepList
   default: []
+- name: EnablePRGeneration
+  type: boolean
+  default: false
+- name: PRMatrix
```

Comment:
I will swap PRMatrix to inheriting MatrixConfigs and pass that on to the Create-PrJobMatrix in a further commit.

Reply:
Ok this is done now.